src/PayPal/Api/Subscription/ShippingAmount.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Subscription;
+
+use Swag\PayPal\PayPal\Api\Common\PayPalStruct;
+
+class ShippingAmount extends PayPalStruct
```

Comment:
extend from https://github.com/shopwareLabs/SwagPayPal/blob/master/src/PayPal/Api/Common/Value.php

Reply:
not possible, see FixedPrice . Perhaps i should make a Subscription/Common/Value file?

Reply:
according to the docs, it is called "money" https://developer.paypal.com/docs/api/subscriptions/v1/#definition-money

so please create a money struct in the global common namespace :+1: